frontend/src/styles/components/MinePermitTable.scss

```diff
@@ -0,0 +1,50 @@
+.permit-table-row {
+  vertical-align: middle;
+  td {
+    vertical-align: middle;
+  }
+}
+
+.icon-svg-filter {
+  filter: brightness(0) saturate(100%) invert(33%) sepia(15%) saturate(2661%) hue-rotate(216deg)
+    brightness(89%) contrast(92%);
+}
+
+.permit-table-button {
+  margin: 0;
+  border: 0;
+  color: $violet;
+  background: none;
+  padding-left: 5px;
+  padding-right: 0px;
+
+  &:hover {
+    border: 0;
+    color: $violet;
+    background: none;
+  }
+
+  div {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+
+    img {
+      min-height: 18px;
+    }
+  }
+}
+
+.add-permit-dropdown-button {
+  div {
+    display: flex;
+    align-items: center;
+    padding-right: 10px;
+  }
+}
+
+.add-permit-dropdown-button-icon {
+  font-size: 18px;
+  padding-left: 8px;
+  padding-right: 20px;
+}
```

Comment:
This file concerns me a little.
All of our buttons have pre-defined CSS or can use a combination of existing classes the meet the desired outcome. 
These existing classes should be used over creating custom CSS for permit-specific buttons. 
Same goes for tables.

Reply:
The drop-down list component being wrapped around buttons has an effect on the rendering of the child button and it's own child parts (image, text). I had to do some specific tweaking to get it to render as one would expect.

Additionally, there are are mixed icon/images in the dropdown that have mixed spacing. They required different padding to align. They are independent components rendered by the dropdown list, so no way to have columns for alignment.

I'm happy moving the permit-table-row class to a common place and naming it generically "center-table-row" or something. There's currently no other class I can see that does this.

Reply:
Okay, Thank you for explaining! 

aaaaand Table.scss would be a good spot for all the table related css!